modules/hostap/Kconfig

```diff
@@ -486,6 +507,15 @@ config NO_RANDOM_POOL
 config WNM
 	bool
 
+config BGSCAN
+	bool
+
+config BGSCAN_SIMPLE
+	bool
+
+config BGSCAN_LEARN
+	bool
```

Comment:
@krish2718 did we come to a conclusion for these?

Reply:
these should be prefixed with hostap, bgscan by itself could refer to anything, it could be submitted as a subsystem and this would just block it just because

Reply:
acceptable on the understanding that these symbols are short lived and WILL be removed because hostap will be reworked

Reply:
Tracking issue https://github.com/zephyrproject-rtos/zephyr/issues/95779